src/code-analysis/comment-collector.js

```diff
@@ -153,3 +156,43 @@ function applyMemberRoles(members, data) {
 
   return result;
 }
+
+/**
+ * @param {Area[]} members
+ */
+function collectExceptions(members) {
+  const result = [];
+
+  for (const member of members) {
+    const _member = { ...member };
+    const { comment } = member;
+
+    if (comment.includes('* @private')) continue;
+
+    if (
+      !comment.includes('* @name ') &&
+      !comment.includes('* @typedef ') &&
+      !comment.includes('* @callback ') &&
+      !comment.includes('* @memberof ') &&
+      !comment.includes('* @memberOf ') &&
+      !comment.includes('* @external ') &&
+      !comment.includes('* @property ') &&
```

Comment:
I suppose we are not looking for property because we don't create full interfaces for all of our classes?

Reply:
The generator is not programmed to do anything with properties, they are ignored.